x-pack/plugins/timelines/public/components/t_grid/integrated/index.tsx

```diff
@@ -292,21 +290,22 @@ const TGridIntegratedComponent: React.FC<TGridIntegratedProps> = ({
       >
         {loading && <EuiProgress size="xs" position="absolute" color="accent" />}
 
+        {graphOverlay}
         {canQueryTimeline ? (
           <>
             <EventsContainerLoading
               data-timeline-id={id}
               data-test-subj={`events-container-loading-${loading}`}
             >
               <UpdatedFlexGroup gutterSize="m" justifyContent="flexEnd" alignItems={alignItems}>
-                <UpdatedFlexItem grow={false} show={!loading}>
+                <UpdatedFlexItem grow={false} $show={!loading}>
                   <InspectButton title={justTitle} inspect={inspect} loading={loading} />
                 </UpdatedFlexItem>
-                <UpdatedFlexItem grow={false} show={!loading}>
+                <UpdatedFlexItem grow={false} $show={!loading}>
                   {!resolverIsShowing(graphEventId) && additionalFilters}
                 </UpdatedFlexItem>
                 {tGridEventRenderedViewEnabled && entityType === 'alerts' && (
-                  <UpdatedFlexItem grow={false} show={!loading}>
+                  <UpdatedFlexItem grow={false} $show={!loading}>
```

Comment:
Good idea, we need this change in x-pack/plugins/timelines/public/components/t_grid/standalone/index.tsx Line 318 and 321 too

Reply:
yep the type check caught that for me too, thanks!!